app/src/main/java/org/oppia/android/app/mydownloads/downloads/SortByItems.kt

```diff
@@ -0,0 +1,12 @@
+package org.oppia.android.app.mydownloads.downloads
+
+/** Enum class containing the items for the Recycler view of [DownloadsFragment]. */
+enum class SortByItems(val value: String) {
+  NEWEST("Newest"),
```

Comment:
Define these strings in `strings.xml` file.

Reply:
done